### DISCUSSION TITLE
Inline Relationships at text editor

### DISCUSSION THREAD

**Discussion by c001os** [directus/directus#15527] [Archive]
Hi!

Im looking to implement a function to the directus wyswyq text editor where i can make inline relationships. I have a great example (sorry another framewrok): https://keystonejs.com/docs/guides/document-fields#inline-relationships
Almost exatly what i want to recreate with directus. Everybody has some hint, or some experience with text editorin directus and inline relations with gui, or another way to link an element from a collection in the text editor?

Thanx a lot!

**Comment by rijkvanzanten** [comment]
The system WYSIWYG editor saves values as MD/HTML, so there's no real way to rely on foreign key constraints / relational nesting inside those blob values. You could take a look at creating a custom interface using something that stores in a formatted value like editor.js 🙂

**Reply by quentinglorieux** [reply]
Wha. I'm very excited by this idea. Any update on this ?
